notesystem/common/visual.py

```diff
@@ -64,3 +66,22 @@ def print_doc_error(doc_errs: DocumentErrors, err_fixed: bool = False) -> None:
                     colored('    Auto fixable:', 'blue'),
                     colored('No', 'red'),
                 )
+
+
+def print_search_result(match, pattern: str) -> None:
```

Comment:
Note: Missing type annotation but it can not be imported because circular imports...